pkg/ddl/backfilling_proto.go

```diff
@@ -20,7 +20,7 @@ import "github.com/pingcap/tidb/pkg/disttask/framework/proto"
 // the initial step is StepInit(-1)
 // steps are processed in the following order:
 // - local sort:
-// StepInit -> StepReadIndex -> StepWriteAndIngest -> StepDone
+// StepInit -> StepReadIndex -> StepDone
```

Comment:
```suggestion
// StepInit -> StepReadIndexAndIngest -> StepDone
```

Reply:
The name is not suitable for global sort:

```
// - global sort:
// StepInit -> StepReadIndexAndIngest -> StepMergeSort -> StepWriteAndIngest -> StepDone
```